source/development/feature-flags.html.md

```diff
@@ -0,0 +1,76 @@
+---
+title: Feature flags in GOV.UK Frontend
+weight: 1
+last_reviewed_on: 2023-11-28
+review_in: 6 months
+---
+# Using feature flags in GOV.UK Frontend
+
+Feature flags are a way for us to release large scale or experimental work early by allowing users to opt in our out of a new, breaking change or feature. Use this guidance to decide if a feature flag is useful for your use case.
+
+## When a feature flag is useful
+Feature flags are useful when the following conditions are true.
+
+### The thing is a breaking change
+Feature flags should only be used for things considered breaking changes. This is something where users will potentially need to change their service, or assess if their service needs changing, in order to continue staying up to date with govuk-frontend. In this case, a feature flag would be a way for a user to stay up to date whilst also opting out of the feature for a period of time. We don’t need to do this for minor releases or new features such as a new component because a user can opt out by simply not using the new feature.
+
+### The thing might be a long term project for users
+When assessing if a breaking change should be behind a feature flag, consider the scope of work required by the user. A feature flag is useful if the change is very difficult to implement and/or has wide scope. Using the green link example in the previous section, this would impact all links in a given service so that service will likely need to check all their pages to ensure the design of their pages still made sense against the new link colour.
```

Comment:
I can't find the green link example. Has it changed name during editing?

Reply:
Oops I removed it, good spot. I'll put it back in and try to make it more concise.

Reply:
Done. Whatcha reckon?